Options.py

```diff
@@ -1428,6 +1434,127 @@ def verify(self, world: typing.Type[World], player_name: str, plando_options: "P
             link["item_pool"] = list(pool)
 
 
+@dataclass(frozen=True)
+class PlandoItem:
+    items: list[str] | dict[str, typing.Any]
+    locations: list[str]
+    world: int | str | bool | None | typing.Iterable[str] | set[int] = False
+    from_pool: bool = True
+    force: bool | typing.Literal["silent"] = "silent"
+    count: int | bool | dict[str, int] = False
+    percentage: int = 100
+
+
+class PlandoItems(Option[typing.List[PlandoItem]]):
+    """Generic items plando."""
+    default = []
```

Comment:
This is something that is talked about a lot in AP discord dev channels.
Mutable defaults are dangerous. A lot of people have run into bugs because of them.
The empty `tuple` is iterable, and both `__init__` and `from_any` work with iterables.
```suggestion
    default = ()
```

Reply:
@Exempt-Medic